src/celeritas/user/detail/SDParams.cc

```diff
@@ -0,0 +1,102 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/user/detail/SDParams.cc
+//---------------------------------------------------------------------------//
+#include "SDParams.hh"
+
+#include "corecel/data/CollectionBuilder.hh"
+#include "corecel/io/Join.hh"
+#include "geocel/GeoVolumeFinder.hh"
+
+namespace celeritas
+{
+
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from list of volume labels.
+ */
+SDParams::SDParams(std::string output_label,
```

Comment:
I don't think this is needed; we'll only have one SD params.

Reply:
Ok, I have removed the output_label, both as an input parameter and a member of `SDParams`